### DISCUSSION TITLE
Schéma causal

### DISCUSSION THREAD

**Discussion by Marine1719** [BIO6032/2022-SEIR-reproduction#4] [General]
Concernant le schéma causal; 
Est-on obligé de reproduire exactement le même schéma causal ?? Car je ne comprends pas leur choix de ne pas mettre de lien de S  vers symptomatic individuals, dans la plupart des articles que j'ai lu, en général c'est scindé en S --> Exposed --> I symptomatic et I asymptomatic donc leur choix de mettre uniquement une flêche de I asymptomatic vers I symptomatic est perturbant sachant qu'une fois devenu infectieux on peut directement avoir des symptômes. Du coup je ne comprends pas bien..

**Comment by blanchette31** [comment]
Je crois que c'est juste un choix que l'auteur a fait de ne pas inclure le compartiment Exposed dans le modèle. Selon l'article, ça semble être inspirée par le modèle dans la source (8). Au lieu d'avoir un compartiment Exposed et ensuite deux branches vers asymptomatique et symptomatique, on passe de susceptible à asymptomatique (à un taux béta) et d'asymptomatique à symptomatique à un taux de conversion s. C'est-à-dire, que si le taux s est 0,4, par exemple, 40% des personnes infectieux développeront des symptômes. À mon avis, ce n'est qu'une autre manière de concevoir le modèle, mais qu'au final, ça revient essentiellement au même. J'espère que ça réponds à ta question.

**Comment by tpoisot** [comment]
Dans ce cas la, ça vaut la peine de tenter de reproduire le diagramme à partir des équations!
